fuzz/fuzz_targets/packet.rs

```diff
@@ -0,0 +1,29 @@
+#![cfg_attr(not(windows), no_main)]
+
+#[cfg(not(windows))]
+use libfuzzer_sys::fuzz_target;
+
+#[cfg(not(windows))]
+fuzz_target!(|data: &[u8]| {
+    use std::sync::Once;
+
+    use neqo_transport::{packet::PublicPacket, RandomConnectionIdGenerator};
+
+    static INIT: Once = Once::new();
+    static mut DECODER: Option<neqo_transport::RandomConnectionIdGenerator> = None;
+
+    // Initialize things
+    unsafe {
+        INIT.call_once(|| {
+            neqo_crypto::init();
```

Comment:
`neqo_crytpo::init` already itself uses a `std::sync::Once`. Is the additional `Once` needed here?

https://github.com/mozilla/neqo/blob/6344d25eaebb9a873e72adb6252901dd9579a763/neqo-crypto/src/lib.rs#L114-L129